passwd/lib/Driver/Ispconfig.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Copyright 2009-2016 Horde LLC (http://www.horde.org/)
+ *
+ * See the enclosed file COPYING for license information (GPL). If you
+ * did not receive this file, see http://www.horde.org/licenses/gpl.
+ *
+ * @category  Horde
+ * @copyright 2009-2016 Horde LLC
+ * @license   http://www.horde.org/licenses/gpl GPL
+ * @package   Passwd
+ */
+/**
+ * Changes a email password using the IspConfig 3 API.
+ *
+ * @author    Thomas Basler <tbasler@oprago.com>

```

Comment:
Since you re-used some code from Ingo, which is perfectly fine, you should attribute Michael Bunk here too though.